bokeh/models/widgets/markups.py

```diff
@@ -76,6 +80,11 @@ class Markup(Widget):
     Raw CSS style declaration. Note this may be web browser dependent.
     """)
 
+    disable_math = Bool(False, help="""
+    Whether the contents should not be processed as TeX/LaTeX input.
+    The default value is False.
```

Comment:
We know what's the default value from the properties' system. There is no need for duplication, unless you want to explain more about the default case.